.github/workflows/python-package.yml

```diff
@@ -5,7 +5,9 @@ name: Python package
 
 on:
   push:
-    branches: [ '**' ]

```

Comment:
Why do we need to remove this ? Is it required to push on master an ignore for the specific branch "TestDockerInGithubActions" ? If not, we can revert this